multiversx_sdk_wallet/crypto/encryptor.py

```diff
@@ -0,0 +1,46 @@
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes, hmac
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.primitives.kdf.scrypt import Scrypt
+
+from multiversx_sdk_wallet.crypto.constants import (CIPHER_ALGORITHM,
+                                                    ENCRYPTOR_VERSION,
+                                                    KEY_DERIVATION_FUNCTION)
+from multiversx_sdk_wallet.crypto.encrypted_data import (EncryptedData,
+                                                         KeyDerivationParams)
+from multiversx_sdk_wallet.interfaces import IRandomness
+
+
+def encrypt(data: bytes, password: str, randomness: IRandomness) -> EncryptedData:
+    """
+    Also see: https://github.com/multiversx/mx-sdk-js-wallet/blob/main/src/crypto/encryptor.ts
+    """
+    backend = default_backend()
+
+    kdParams = KeyDerivationParams(n=4096, r=8, p=1, dklen=32)
+    kdf = Scrypt(salt=randomness.salt, length=kdParams.dklen, n=kdParams.n, r=kdParams.r, p=kdParams.p, backend=backend)
```

Comment:
Like above, this line assumes that `KEY_DERIVATION_FUNCTION == "scrypt"`. Which is true at the moment, of course, but it could be changed at some point, maybe?

Reply:
Renamed constants. If we change this at some point, the JavaScript SDK will be first updated, and then we'll replicate the conventions and design from there.